js/average.js

```diff
@@ -0,0 +1,46 @@
+
+function total(array){
+    var total = array.reduce(function (accumulator, currentValue) {
+        return accumulator + currentValue;
+      }, 0);
+      return total
+}
+
+function mean(array){
+    var sum = total(array);
+    return sum/(array.length)
+}
+
+function median(array){
```

Comment:
Nice re-use of `mean`! 

Watch your indentation/spacing here for readability. Let me know if you're not sure what would change here and we can go through it.